lib/contributor.rb

```diff
@@ -0,0 +1,11 @@
+class Contributor
+  def initialize(contributor)
+    @contributor = contributor
+  end
+
+  def add
+    File.open('CONTRIBUTORS.md', 'a+') do |f|
+      f.puts "- [@#{@contributor}] (https://github.com/#{@contributor})"
+    end
+  end
+end
```

Comment:
@rafaelsales You're right, I started playing this way with `File` creating a temp.md but for some reason file reference was not changing so testing always broke. I'll try it with Tempfile.

@rthbound Haven't consider alphanumeric.